<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jeferson Coli</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>924</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tecnocoli  -  Auto Diagnostic Scanner and DataLogger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>801</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
alternate-background-color: rgb(36, 7, 255);
</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Principal</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1071</width>
        <height>751</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>118</width>
         <height>27</height>
        </rect>
       </property>
       <property name="displayFormat">
        <string>hh:mm </string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>110</width>
         <height>27</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDate|Qt::ImhTime</set>
       </property>
       <property name="displayFormat">
        <string>d/M/yy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>1051</width>
         <height>521</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(250, 250, 250);
</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextEditorInteraction</set>
       </property>
      </widget>
      <widget class="QPushButton" name="ReconnectButton">
       <property name="geometry">
        <rect>
         <x>850</x>
         <y>710</y>
         <width>87</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Connect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExit">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>710</y>
         <width>87</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>E&amp;xit</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdRpm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>132</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 102, 102);
color: rgb(174, 255, 242);
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdSpeed">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>132</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 102, 102);
color: rgb(174, 255, 242);
</string>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdCTemp">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>132</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 102, 102);
color: rgb(174, 255, 242);
</string>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>143</x>
         <y>140</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>RPM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>333</x>
         <y>140</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>km/h</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>523</x>
         <y>140</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Â°C</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_scan_odb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>710</y>
         <width>87</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Scan</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_mil_on">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>110</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../OBD2-scanner-datalogger-1/venv/images/mil-on.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_mil_off">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>110</y>
         <width>61</width>
         <height>51</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:20px</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../OBD2-scanner-datalogger-1/venv/images/mil-off.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_dtc">
     <attribute name="title">
      <string>DTC</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_dtc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>1051</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 250, 250);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReadDtc">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Read DTC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearDtc">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>30</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear DTC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Import Data</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>321</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QQuickWidget" name="quickWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>300</width>
        <height>200</height>
       </rect>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_data_card">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTableWidget" name="table_data_card">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1041</width>
        <height>721</height>
       </rect>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_graphic">
     <attribute name="title">
      <string>Graphics</string>
     </attribute>
     <widget class="GraphWidget" name="GraphWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>500</width>
        <height>350</height>
       </rect>
      </property>
     </widget>
     <widget class="GraphWidget2" name="GraphWidget2" native="true">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>500</width>
        <height>350</height>
       </rect>
      </property>
     </widget>
     <widget class="GraphWidget3" name="GraphWidget3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>500</width>
        <height>350</height>
       </rect>
      </property>
     </widget>
     <widget class="GraphWidget4" name="GraphWidget4" native="true">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>370</y>
        <width>500</width>
        <height>350</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_up_graph">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>730</y>
        <width>87</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Graph</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_gauges">
     <attribute name="title">
      <string>Gauges</string>
     </attribute>
     <widget class="GaugeRpmWidget" name="GaugeRpmWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>511</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Configurations</string>
     </attribute>
     <widget class="QPushButton" name="btn_test_obd2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>720</y>
        <width>87</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Test OBD2</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>1091</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLCDNumber" name="lcdTime">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>4</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Mono</family>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>8</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>27</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save"/>
    <addaction name="action_Open"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QStatusBar" name="status_bar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 190, 255);</string>
   </property>
  </widget>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>graphwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GaugeRpmWidget</class>
   <extends>QWidget</extends>
   <header>gaugerpmwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphWidget2</class>
   <extends>QWidget</extends>
   <header>graphwidget2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphWidget3</class>
   <extends>QWidget</extends>
   <header>graphwidget3.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphWidget4</class>
   <extends>QWidget</extends>
   <header>graphwidget4.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
